src/TriggerBinding/SqlTriggerBinding.cs

```diff
@@ -78,8 +78,9 @@ public async Task<IListener> CreateListenerAsync(ListenerFactoryContext context)
         {
             _ = context ?? throw new ArgumentNullException(nameof(context), "Missing listener context");
 
-            string userFunctionId = await this.GetUserFunctionIdAsync();
-            return new SqlTriggerListener<T>(this._connectionString, this._tableName, this._leasesTableName, userFunctionId, context.Executor, this._sqlOptions, this._logger, this._configuration);
+            string userFunctionId = this.GetUserFunctionId();
```

Comment:
Even after a migration is done once, are we still calculating these old + new values? Is that really needed?

Reply:
Yes, we decided to have the queries such that if old values are present, migrate them and clean up else continue with new values.